Murcielago/Murcielago/UserInterface/Views/SettingsView/BatteryIndicatorView.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  BatteryIndicatorView.swift
+//  Murcielago
+//
+//  Created by Sam Franusic on 12/3/24.
```

Comment:
Nitpick on dates: I prefer [ISO 8601](https://en.wikipedia.org/wiki/ISO_8601) dates as they are unambiguous.

Reply:
I will consider doing this. This can be done by updating Xcode's template header file